builder/azure/arm/azure_client.go

```diff
@@ -281,7 +281,7 @@ func buildAuthorizer(ctx context.Context, authOpts NewSDKAuthOptions, env enviro
 	var authConfig auth.Credentials
 	switch authOpts.AuthType {
 	case AuthTypeDeviceLogin:
-		return nil, fmt.Errorf("DeviceLogin is not supported, however you can use the Azure CLI `az login --use-device-code` to use a device code, and then use CLI authentication")
+		return nil, fmt.Errorf("DeviceLogin is not supported in v2 of the Azure Packer Plugin, however you can use the Azure CLI `az login --use-device-code` to use a device code, and then use CLI authentication")
```

Comment:
Are these changes from another branch? Or do you need to rebase onto main. These changes pertain to the arm builder and not DTL.

Reply:
While adding the `dtl/azure_client` I thought it would be better to clarify this error message, I made the same change on the ARM client for consistency